### PR TITLE
Lesson 1.3

### PR DESCRIPTION
**Pull request by Jay-zar**
Lesson 1.3 homework

### CODE REVIEW

File: src/addToDoForm.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+        console.log(toDoTitle);
+        event.target.title.value= '';
+        props.onAddToDo(toDoTitle);
+    };
     return(
-        <form>
-            <label htmlFor="Title">Title</label> <input id= "Search"></input><button>Add</button>
+        <form onSubmit={handleAddToDo}>
+            <label htmlFor="Title">Title</label> 
+            <input id= "Search" type= "text" name= "title" />
```
Comment (jkylehumphrey) [inline src/addToDoForm.js:15]: proper formatting would remove the spaces here.  All markup should be like:
`type="text"`

File: src/App.js [hunk 2]
```diff
@@ -3,10 +3,12 @@ import AddToDoForm from './addToDoForm';
 import ToDoList from './todolist';
 
 function App() {
+  const [newToDo, setNewToDo]= React.useState('');
   return (
     <div style={{ textAlign: 'center' }}>
      <h1>To Do List</h1>
-     <AddToDoForm/>
+     <AddToDoForm onAddToDo= {setNewToDo}/>
```
Comment (jkylehumphrey) [inline src/App.js:10]: remove spaces:
`onAddToDo={setNewToDo}`

**Review by jkylehumphrey — CHANGES_REQUESTED** [review]
Looks good!  Just a few formatting changes

File: src/addToDoForm.js (hunk 1 above)
Comment (Jay-zar) [reply src/addToDoForm.js:15]: I see, thank you for the corrections! I will be more consistent on that going forward.

**Pull request merged**